assets/constants.js

```diff
@@ -540,6 +540,18 @@ var constants = {
 
   "NUM_QUESTIONS_PER_PAGE": 10,
 
+  "PROBABILITY_INDEXES" : {
+      // The probabilty index when the outcome is not equal to the default outcome
```

Comment:
Wait, isn't the highest probability for when the outcome does equal the default outcome, I don't think the comment is right.

Reply:
Okay

Reply:
Done